docs/.vitepress/config/sidebar/apiSidebar.json

```diff
@@ -0,0 +1 @@
+{"text": "API Reference", "items": [{"text": "config modules", "link": "api/starlark/config"}, {"text": "install modules", "link": "api/starlark/install"}, {"text": "api modules", "link": "api/starlark/api"}, {"text": "config modules", "link": "api/starlark/config"}, {"text": "install modules", "link": "api/starlark/install"}, {"text": "runtime modules", "link": "api/starlark/runtime"}, {"text": "io modules", "link": "api/starlark/io"}, {"text": "global functions", "link": "api/starlark/global_functions"}]}
```

Comment:
Is it generated file? Two `config` and `install`

Reply:
Fixed. Due to legacy code